runtime/compiler/trj9/control/J9Options.cpp

```diff
@@ -85,6 +85,9 @@ int32_t J9::Options::_smallMethodBytecodeSizeThresholdForCold = -1; // -1 means
 
 int32_t J9::Options::_countForMethodsCompiledDuringStartup = 10;
 
+int32_t J9::Options::_countForLoopyBootstrapMethods = 3000; // 3000 means disable the feature by default
+int32_t J9::Options::_countForLooplessBootstrapMethods = 3000; // means disable the feature by default
```

Comment:
Why does 3000 mean disabled? I'm assuming it's because the default count and bcount are also 3000. The symbolic constants for those values are `TR_DEFAULT_INITIAL_COUNT` and `TR_DEFAULT_INITIAL_BCOUNT` but even then it looks like the actual initial count we go with is determined at run time based on other options (e.g. quickstart), so if the goal is to keep these changes disabled for now by keeping the behaviour the same this might not be sufficient. It seems like these counts shouldn't be statically initialized, they should be initialized with whatever values we decide on for the default counts.

Reply:
Yeah, I was loose here because I am about to change them to 1000/250 anyway. I'll do better.

Reply:
I changed the code to set the initial value of `_countForLoopyBootstrapMethods` and  `_countForLooplessBootstrapMethods` to -1. This will enable the old code path to be taken.